src/components/profile/Recovery/RecoveryContainer.js

```diff
@@ -82,7 +82,7 @@ const RecoveryContainer = () => {
     const dispatch = useDispatch();
     const account = useSelector(({ account }) => account);
     const accountId = account.accountId;
-    const activeMethods = useRecoveryMethods(accountId).filter(method => method.confirmed);
+    const activeMethods = useRecoveryMethods(accountId).filter(method => method.publicKey);
```

Comment:
@Patrick1904 shouldn't we check `method.confirmed && method.publicKey`? Or am I misunderstanding smth?

Reply:
I checked `method.publicKey` since we only add a `publicKey` once a method is confirmed, but that might change. I updated to check `method.confirmed` too.